matrixbot/matrix.py

```diff
@@ -36,15 +36,25 @@ def __init__(self, settings):
         self.only_local_domain = matrix["only_local_domain"]
         self.super_users = matrix.get("super_users", [])
 
-        self.commands_enable = settings.get("commands", {}).get("enable", True)
+        self.commands_enable = settings\
+            .get("commands", {})\
+            .get("enable", True)
 
-        self.subscriptions_room_ids = list(settings["subscriptions"].keys())
-        self.revokations_rooms_ids = list(settings["revokations"].keys())
+        self.subscriptions_room_ids = settings.get("subscriptions", {}).keys()
+        self.revokations_rooms_ids = settings.get("revokations", {}).keys()
```

Comment:
I would split lines 43 and 44 to match style around. Breaking before the dot and keeping one get per line.